src/docfx/build/manifest/DependencyMap.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Docs.Build
+{
+    internal class DependencyMap
```

Comment:
This could be just `List<DpenendyItem>`, why do we care `Dependencies` vs `InclusionDependencies` here? You could derive whether it is include from `DependencyType`

Reply:
InvlusionDependencies means the inclusions' dependencies, otherwise the token or codesnippet's dependencies will be lost

---
In reply to: [193675368](https://github.com/dotnet/docfx/pull/2859#discussion_r193675368) [](ancestors = 193675368)